web/vital_records/models.py

```diff
@@ -18,8 +18,6 @@ class VitalRecordsRequest(models.Model):
         ("finished", "Finished"),
     ]
 
-    NUMBER_CHOICES = [(1, "1"), (2, "2"), (3, "3"), (4, "4"), (5, "5"), (6, "6"), (7, "7"), (8, "8"), (9, "9"), (10, "10")]
-
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     status = FSMField(default="initialized", choices=STATUS_CHOICES)
```

Comment:
Good callout that this still has `choices` on it @lalver1.

It seems we could remove this `choices` as well and therefore also remove `STATUS_CHOICES`. The point that led to us deciding to remove `choices` from the model fields was that we're not actually using any sort of bulit-in, default implementation from Django that is driven by them, and that applies here too.

Let me know what you think @lalver1 @thekaveman

Reply:
You're right @thekaveman :+1:  Looks like `FSMField` does require us to set `choices` - see [`FSMFieldMixin` init logic](https://github.com/django-commons/django-fsm-2/blob/ff12c88527559a83654a1b728152cede5c711eb1/django_fsm/__init__.py#L238-L239).